packages/react-composites/tests/browser/call/UFD.test.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from '@playwright/test';
+import { dataUiId, clickOutsideOfPage, waitForPageFontsLoaded, waitForSelector } from '../common/utils';
+import { test } from './fixture';
+import { buildUrlWithMockAdapter } from './utils';
+import { DiagnosticQuality } from './TestCallingState';
+
+test.describe('UFD tests', async () => {
+  test('UFD test when speaking while muted ', async ({ pages, serverUrl }) => {
```

Comment:
nit: Describe what the test does "A banner is shown when user is speaking while muted"

And same for other tests. These test descriptions should be sensible when reading on the console. They should give us a reasonable idea of what failed without looking at the test code.